app/gui/src/project-view/components/DockPanel.vue

```diff
@@ -140,6 +140,10 @@ const tabStyle = {
   }
 }
 
+.tab .ToggleIcon svg {
+  opacity: 1 !important;
+}
```

Comment:
If we don't display toggle state, we could just replace ToggleIcon with SvgButton component.

Reply:
Done